CSSIntro.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let button = document.querySelector("button");
+  button.addEventListener("click", () => {
+    let CSSfile = "./blank.css"
+    getCrrLink = document.querySelector("link");
+
+    newLink = document.createElement("link");
+    newLink.setAttribute("rel", "stylesheet");
+    newLink.setAttribute("href", CSSfile);
+    document.querySelector("head").replaceChild(newLink, getCrrLink);
```

Comment:
This looks good, but try to make a button that removes it and adds the css back again.